mcc/PrintLineWithStyles.c

```diff
@@ -491,26 +529,26 @@ LONG PrintLine(struct InstData *data, LONG x, struct line_node *line, LONG line_
     {
       if(line_nr == 1)
       {
-        BltBitMapRastPort(rp->BitMap, xoffset, _mtop(data->object)-data->ypos, data->rport, _mleft(data->object), _mtop(data->object)+(data->fontheight * (line_nr-1)), _mwidth(data->object), data->fontheight-(_mtop(data->object)-data->ypos), (ABC|ABNC));
+        BltBitMapRastPort(rp->BitMap, 0, _mtop(data->object)-data->ypos, data->rport, _mleft(data->object), _mtop(data->object)+(data->fontheight * (line_nr-1)), _mwidth(data->object), data->fontheight-(_mtop(data->object)-data->ypos), (ABC|ABNC));
```

Comment:
Actually using xoffset value there was a **very very wrong** logic to begin with.

The blit copy from BitMap is **ONLY** done when in **doublebuffer** mode.
And when in doublebuffer mode, the **xoffset** value is always **0** (please see the lines **140** and **168** of **mcc/PrintLineWithStyles.c** in your repository) which makes sense because we should always be copying from the very start of the buffering bitmap. 

It should have been 0 to begin with.